docs/source/spawners.md

```diff
@@ -163,4 +163,6 @@ When `Spawner.start` is called, this dictionary is accessible as `self.user_opti
 [Spawner]: https://github.com/jupyter/jupyterhub/blob/master/jupyterhub/spawner.py
 
 
-## Writing a custom spawner
\ No newline at end of file
+## Writing a custom spawner
+
+If you are interested in building a custom spawner, you can read [this tutorial](http://jupyterhub-tutorial.readthedocs.io/en/latest/spawners.html)
```

Comment:
Hi @scrypy, I'm going to merge as is and then you can open a new PR to take care of the period at the end of the sentence. Yay!

Thanks and you can check your remote settings with `git remote -v`. 

You may find this talk on git that I did in the past helpful: http://www.slideshare.net/willingc/yes-you-can-git